ex_standards/ex_http_status.md

```diff
@@ -9,34 +9,34 @@ Some might ok with 2 options, choose one and justify it.
 ----
 
 * I cannot access that web if I'm not logged:
-  *  
+  *  401 Unauthorized
 
 * I'm logged but I cannot access that page
-  * 
+  * 403 Forbidden
 
 * The item with that Id was deleted
-  * 
+  * 410 Gone
 
 * I started uploading, but I have to check another link to see if it is there
   * 
 
 * That query parameter is wrong
-  * 
+  * 400 Bad Request
 
 * There are no resources with that ID
-  * 
+  * 404 Not Found
 
 * My request was good, but it will take time to resolve
-  * 
+  * 425 Too Early
```

Comment:
Aquí estaba buscando un 202 Accepted (to process the request). É o máis habitual para procesos asíncronos. Pero 425 podería ser unha opción léxitima (primeira vez que o vexo)